src/img/glyph-pin-pinned-12.svg

```diff
@@ -4,6 +4,9 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="12" height="12" viewBox="0 0 12 12">
   <style>
+    .icon:not(:target) {
```

Comment:
Do we need these?

Reply:
They won't render properly without it. It will try to render both < use > tags on top of each other. It's the same CSS that tab-audio-small.svg uses to store multiple icons.

Reply:
Ha that's fine thanks!